packages/ra-test/README.md

```diff
@@ -0,0 +1,176 @@
+# ra-test
+
+Test utilities for react-admin, a frontend Framework for building admin applications on top of REST services, using ES6, React.
+
+By default, react-admin acts as a declarative admin configuration: list some resources, define their controllers and, plug some built-in components or your own to define their fields or inputs.
```

Comment:
```suggestion
By default, react-admin acts as a declarative admin configuration: list some resources, define their controllers, and,plug some built-in components or your own to define their fields or inputs.
```

Reply:
You know I copy/pasted our UnitTesting documentation right ?